src/vcpkg/build.cpp

```diff
@@ -1711,6 +1719,9 @@ namespace vcpkg
                 case VcpkgTripletVar::PUBLIC_ABI_OVERRIDE:
                     public_abi_override = variable_value.empty() ? nullopt : Optional<std::string>{variable_value};
                     break;
+                case VcpkgTripletVar::HASH_ADDITIONAL_FILES:
+                    Util::Vectors::append(&hash_additional_files, Strings::split(variable_value, ';'));
```

Comment:
```suggestion
                    hash_additional_files = Strings::split(variable_value, ';');
```
This can only be triggered once for a triplet.